protocol/daemons/server/liquidation.go

```diff
@@ -47,12 +37,7 @@ func (s *Server) LiquidateSubaccounts(
 		metrics.Received,
 		metrics.Count,
 	)
-	// If the daemon is unable to report a response, there is either an error in the registration of
-	// this daemon, or another one. In either case, the protocol should panic.
-	if err := s.reportResponse(types.LiquidationsDaemonServiceName); err != nil {
-		s.logger.Error("Failed to report liquidations response to update monitor", "error", err)
-	}
-
+	s.reportResponse(types.LiquidationsDaemonServiceName)
```

Comment:
For both `ModuleSetGauge` and `reportResponse`: 

Given that we don't know if the below logic will succeed or not at this point, should we move these to right before the return statement?

Reply:
I think additional feature work here is out of scope for this PR.